.azure-pipelines/integration-tests.yml

```diff
@@ -6,15 +6,6 @@ trigger:
     exclude:
       - docs/*
       - .github/*
```

Comment:
There seemed to be an issue with PR's being automatically triggered recently, including this one. After removing this (which we can re-introduce in a later PR), this PR is now automatically triggering integration-test pipelines

Reply:
I removed these changes so hopefully everything still triggers okay